src/replication/replication_slave_node.cpp

```diff
@@ -123,9 +123,6 @@ namespace cubreplication
     delete g_instance->m_lc;
     g_instance->m_lc = NULL;
 
-    delete g_instance->m_stream;
```

Comment:
There is no need to delete explicitly m_stream here since m_stream will be deleted in parent destructor of g_instance class